### ISSUE TITLE
Error: Need Better Formatting

### ISSUE THREAD

**Issue by mcwhittemore** [mcwhittemore/bughub#5]
## Frame 7: [/Users/matthewwhittemore/mcw/bughub/test/run.js @ 11](https://github.com/mcwhittemore/bughub/blob/master/test/run.js#L11)
```js

var raven = require('raven');
var client = new raven.Client('http://'+process.env.GITHUB_USERNAME+':'+process.env.USER_PROJECT_SECRET+'@localhost:3000/sentry/'+process.env.PROJECT_ID);

client.on("error", function(err){ console.log(err); });
client.on("logged", function(msg){ console.log(msg); });
 
error(client, "Need Better Formatting");


```

## Frame 8: [/Users/matthewwhittemore/mcw/bughub/test/error.js @ 2](https://github.com/mcwhittemore/bughub/blob/master/test/error.js#L2)
```js
module.exports = function(client, msg){
	client.captureError(new Error(msg));
}
```

**Comment by mcwhittemore** [comment]
## Details
* server_name: STs-MBP-2.home
* culprit: error at module.exports
* platform: node
* timestamp: 2015-04-19T03:59:50
## Stacktrace
### Frame 7: [/Users/matthewwhittemore/mcw/bughub/test/run.js @ 11](https://github.com/mcwhittemore/bughub/blob/master/test/run.js#L11)
```js

var raven = require('raven');
var client = new raven.Client('http://'+process.env.GITHUB_USERNAME+':'+process.env.USER_PROJECT_SECRET+'@localhost:3000/sentry/'+process.env.PROJECT_ID);

client.on("error", function(err){ console.log(err); });
client.on("logged", function(msg){ console.log(msg); });
 
error(client, "Need Better Formatting");


```

### Frame 8: [/Users/matthewwhittemore/mcw/bughub/test/error.js @ 2](https://github.com/mcwhittemore/bughub/blob/master/test/error.js#L2)
```js
module.exports = function(client, msg){
	client.captureError(new Error(msg));
}
```

## Modules
```json
{
  "dotenv": "1.1.0",
  "raven": "0.7.3"
}
```

**Comment by mcwhittemore** [comment]
## Details
* server_name: STs-MBP-2.home
* culprit: error at module.exports
* platform: node
* timestamp: 2015-04-19T04:01:04

## Stacktrace

### Frame 7: [/Users/matthewwhittemore/mcw/bughub/test/run.js @ 11](https://github.com/mcwhittemore/bughub/blob/master/test/run.js#L11)
```js

var raven = require('raven');
var client = new raven.Client('http://'+process.env.GITHUB_USERNAME+':'+process.env.USER_PROJECT_SECRET+'@localhost:3000/sentry/'+process.env.PROJECT_ID);

client.on("error", function(err){ console.log(err); });
client.on("logged", function(msg){ console.log(msg); });
 
error(client, "Need Better Formatting");


```

### Frame 8: [/Users/matthewwhittemore/mcw/bughub/test/error.js @ 2](https://github.com/mcwhittemore/bughub/blob/master/test/error.js#L2)
```js
module.exports = function(client, msg){
	client.captureError(new Error(msg));
}
```

## Modules
```json
{
  "dotenv": "1.1.0",
  "raven": "0.7.3"
}
```

**Comment by mcwhittemore** [comment]
## Details
* server_name: STs-MBP-2.home
* culprit: error at module.exports
* platform: node
* timestamp: 2015-04-19T04:02:02

## Stacktrace

### Frame 7: [/test/run.js @ Line 11](https://github.com/mcwhittemore/bughub/blob/master/test/run.js#L11)
```js

var raven = require('raven');
var client = new raven.Client('http://'+process.env.GITHUB_USERNAME+':'+process.env.USER_PROJECT_SECRET+'@localhost:3000/sentry/'+process.env.PROJECT_ID);

client.on("error", function(err){ console.log(err); });
client.on("logged", function(msg){ console.log(msg); });
 
error(client, "Need Better Formatting");


```

### Frame 8: [/test/error.js @ Line 2](https://github.com/mcwhittemore/bughub/blob/master/test/error.js#L2)
```js
module.exports = function(client, msg){
	client.captureError(new Error(msg));
}
```

## Modules
```json
{
  "dotenv": "1.1.0",
  "raven": "0.7.3"
}
```

**Comment by mcwhittemore** [comment]
## Details
* server_name: STs-MBP-2.home
* culprit: error at module.exports
* platform: node
* timestamp: <time datetime='2015-04-19T04:04:51Z' is='relative-time' title='2015-04-19T04:04:51'>2015-04-19T04:04:51</time>

## Stacktrace

### Frame 7: [/test/run.js @ Line 11](https://github.com/mcwhittemore/bughub/blob/master/test/run.js#L11)
```js

var raven = require('raven');
var client = new raven.Client('http://'+process.env.GITHUB_USERNAME+':'+process.env.USER_PROJECT_SECRET+'@localhost:3000/sentry/'+process.env.PROJECT_ID);

client.on("error", function(err){ console.log(err); });
client.on("logged", function(msg){ console.log(msg); });
 
error(client, "Need Better Formatting");


```

### Frame 8: [/test/error.js @ Line 2](https://github.com/mcwhittemore/bughub/blob/master/test/error.js#L2)
```js
module.exports = function(client, msg){
	client.captureError(new Error(msg));
}
```

## Modules
```json
{
  "dotenv": "1.1.0",
  "raven": "0.7.3"
}
```

**Comment by mcwhittemore** [comment]
## Details
* server_name: STs-MBP-2.home
* culprit: error at module.exports
* platform: node
* timestamp: 2015-04-19T04:06:08

<time datetime='2015-04-19T04:06:08Z' is='relative-time' title='2015-04-19T04:06:08'>2015-04-19T04:06:08</time>

## Stacktrace

### Frame 7: [/test/run.js @ Line 11](https://github.com/mcwhittemore/bughub/blob/master/test/run.js#L11)
```js

var raven = require('raven');
var client = new raven.Client('http://'+process.env.GITHUB_USERNAME+':'+process.env.USER_PROJECT_SECRET+'@localhost:3000/sentry/'+process.env.PROJECT_ID);

client.on("error", function(err){ console.log(err); });
client.on("logged", function(msg){ console.log(msg); });
 
error(client, "Need Better Formatting");


```

### Frame 8: [/test/error.js @ Line 2](https://github.com/mcwhittemore/bughub/blob/master/test/error.js#L2)
```js
module.exports = function(client, msg){
	client.captureError(new Error(msg));
}
```

## Modules
```json
{
  "dotenv": "1.1.0",
  "raven": "0.7.3"
}
```

**Comment by mcwhittemore** [comment]
## Details
* server_name: STs-MBP-2.home
* culprit: error at module.exports
* platform: node
* timestamp: 2015-04-19T04:07:03

<time datetime='2015-04-19T04:07:03Z' is='relative-time' title='2015-04-19T04:07:03'>2015-04-19T04:07:03</time>

## Stacktrace

### Frame 7: [/test/run.js @ Line 11](https://github.com/mcwhittemore/bughub/blob/master/test/run.js#L11)
```js

var raven = require('raven');
var client = new raven.Client('http://'+process.env.GITHUB_USERNAME+':'+process.env.USER_PROJECT_SECRET+'@localhost:3000/sentry/'+process.env.PROJECT_ID);

client.on("error", function(err){ console.log(err); });
client.on("logged", function(msg){ console.log(msg); });
 
error(client, "Need Better Formatting");


```

### Frame 8: [/test/error.js @ Line 2](https://github.com/mcwhittemore/bughub/blob/master/test/error.js#L2)
```js
module.exports = function(client, msg){
	client.captureError(new Error(msg));
}
```

## Modules
```json
{
  "dotenv": "1.1.0",
  "raven": "0.7.3"
}
```

**Comment by mcwhittemore** [comment]
Fixed by 4785df0725ea99ae6567a5b7ad30ef87fac15a3d

**Comment by mcwhittemore** [comment]
## Details
* server_name: STs-MBP-2.home
* culprit: error at module.exports
* platform: node
* timestamp: 2015-04-19T04:25:34

## Stacktrace

### Frame 7: [/test/run.js @ Line 11](https://github.com/mcwhittemore/bughub/blob/master/test/run.js#L11)
```js

var raven = require('raven');
var client = new raven.Client('http://'+process.env.GITHUB_USERNAME+':'+process.env.USER_PROJECT_SECRET+'@localhost:3000/sentry/'+process.env.PROJECT_ID);

client.on("error", function(err){ console.log(err); });
client.on("logged", function(msg){ console.log(msg); });
 
error(client, "Need Better Formatting");


```

### Frame 8: [/test/error.js @ Line 2](https://github.com/mcwhittemore/bughub/blob/master/test/error.js#L2)
```js
module.exports = function(client, msg){
	client.captureError(new Error(msg));
}
```

## Modules
```json
{
  "dotenv": "1.1.0",
  "raven": "0.7.3"
}
```
